Taskfile.yml

```diff
@@ -33,9 +33,21 @@ tasks:
 
   test:
     desc: Run tests
+    cmds:
+      - task: test-unit
+      - task: test-integration
+
+  test-unit:
+    desc: Run unit tests
     cmds:
       - go test -short -race -run '.*' {{ default "-v" .GOFLAGS }} -coverprofile=coverage_unit.txt ./... {{.TEST_LDFLAGS}}
 
+  test-integration:
+    desc: Run integration tests
+    cmds:
+      - poetry install --no-root
+      - poetry run pytest test
+
```

Comment:
Let's build the tool before running tests, it happens way to often that tests fail locally because I forget to build first. :facepalm: 

Per has done something similar on the library engine:
https://github.com/arduino/libraries-repository-engine/blob/96288c23a1d0a1fb79cd676a7c04f95711c55f30/Taskfile.yml#L41-L46